### ISSUE TITLE
Problem installing on windows

### ISSUE THREAD

**Issue by dfcreative** [component/classes#24]
https://github.com/component/dialog/issues/29#issuecomment-107437997

**Comment by stephenmathieson** [comment]
?

**Comment by dfcreative** [comment]
That is related to [classes-component](https://www.npmjs.com/package/classes-component) package in npm, which seems to be an older version of that component. The actual one is [component-classes](https://www.npmjs.com/package/component-classes).

It might be a good idea to deprecate or unpublish _classes-component_, as it causes error installing it.

**Comment by stephenmathieson** [comment]
looks like only @tj can do anything about deprecating it.

quick fix: don't install that module; install the working one instead. ;)

**Comment by dfcreative** [comment]
Yeah. The issue was with the dialog-component - it automatically tried to install the classes-component. It does not anymore.
